kurtosis/src/services/blutgang/config.toml.tmpl

```diff
@@ -24,7 +24,7 @@ expected_block_time = 13000
 # Time between health checks in ms
 health_check_ttl = 400
 # Suppress the health check running info messages
-supress_rpc_check = false
+suppress_rpc_check = false
```

Comment:
According to `blutgang` [code](https://github.com/rainshowerLabs/blutgang/blob/f3e42b3e089d38d2e6fa893f2b92326a53739429/src/config/types.rs#L70) `supress_rpc_check` is spelled correctly